pushServices/src/main/java/com/computerrock/pushServices/PushServiceManager.kt

```diff
@@ -3,6 +3,7 @@ package com.computerrock.pushServices
 internal object PushServiceManager {
 
     private val observers: MutableList<PushServiceObserver> = mutableListOf()
+    private var iPushTokenService: IPushTokenService? = TODO()
```

Comment:
This should be null instead of TODO?
Or actually it should be lateinit, since you will surely get it, if I understood correctly.

Reply:
Fixed